components/registry/org.wso2.carbon.registry.indexing/src/main/java/org/wso2/carbon/registry/indexing/AsyncIndexer.java

```diff
@@ -181,13 +182,27 @@ protected void uploadFiles(List<IndexingTask> tasks) throws RegistryException {
         }
     }
 
+    /**
+     * This class is used to create index document for the indexing tasks submitted by indexing manager. This will
+     * run in multi threaded environment and create index documents for resources submitted for indexing from a
+     * blocking queue and submit them to SolR client.
+     */
     protected static class IndexingTask implements Runnable {
+
+        public static final String REGISTRY_RESOURCE_SYMLINK_PATH = "registry.resource.symlink.path";
```

Comment:
Hi Dinusha,
Thanks for pointing this out. I have updates the same PR with the change.

Thanks